html/who-is-tec.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../css/style-who-is-tec.css">
+    <link href="https://fonts.googleapis.com/css2?family=Rowdies:wght@300;400;700&display=swap" rel="stylesheet">
+    <title>Who Is Tec</title>
+</head>
+<body>
+    <div class="container">
+
+        <!-- Header Section  -->
+
+        <header class="container-header">
+
+            <img src="../Assets/logo.png" alt="logo" class="container-header-logo">
+            <a href="../index.html"><img src="../Assets/home.png" alt="home icon" class="container-header-home"></a>
+    
+        </header>
+
+        <!-- Content Section -->
+
+        <div class="container-content">
+
+            <img src="../Assets/robot.png" alt="robot">
+
+            <p class="container-content-introduction">
+                Hi Friends, Iam Tec. Your new English instructor.<br>
+                Iam happy to teach you how to pronounce English<br>
+                words correctly and know their meanings.
+            </p>
+
+        </div>
+
+        <!-- Speak Section -->
+
+        <div class="container-speak">
+
+            <p class="container-speak-paragraph"> You can call me just Tec </p>
+            <img src="../Assets/volume.png" alt="speak icon" id="volume"> 
+
+        </div>
```

Comment:
**As the previous comment :** 

**Use the semantic element `section` instead of `div`**